Updates for autoscaling

danieltprice opened the pull request:
Updates for Autoscaling:
- Create your first project:
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/get-started-with-neon/setting-up-a-project
- Compute size configuration (Create a project)
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/manage/projects#compute-size-configuration
- Compute size configuration (Edit a compute endpoint)
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/manage/endpoints#compute-size-configuration
- Various changes to create project steps in other topics

duskpoet commented on content/docs/get-started-with-neon/setting-up-a-project.md:
```diff
@@ -10,11 +10,13 @@ To create your first project:
 ![Create a project](/docs/get-started-with-neon/create_project.png)
 
-3. On the **Project Creation** dialog, Specify a name, a PostgreSQL version, and a region, and click **Create project**.
+3. On the **Project Creation** dialog, specify a name, a PostgreSQL version, and a region. If you are a [Neon paid plan](https://neon.tech/docs/introduction/billing#neon-plans) user, you can also specify compute size settings. Neon supports fixed size computes and Autoscaling. For more information about these options, see [Compute size configuration](/docs/manage/projects#compute-size-configurati ...
```
```suggestion
3. On the **Project Creation** dialog, specify a name, a PostgreSQL version, and a region. If you are a [Neon paid plan](https://neon.tech/docs/introduction/billing#neon-plans) user, you can also specify compute size settings. Neon supports fixed size computes and Autoscaling (beta). For more information about these options, see [Compute size configuration](/docs/manage/projects#compute-size-configuration).
```

duskpoet commented on content/docs/manage/projects.md:
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
+
+Neon [paid plan](/docs/introduction/billing#neon-plans) users can configure compute size settings when [creating a project](#create-a-project).
+
+_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 3 CUs. Larger compute sizes will be supported in a future release.
```
```suggestion
_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 7 CUs. Larger compute sizes will be supported in a future release.
```

duskpoet commented on content/docs/manage/projects.md:
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
+
+Neon supports two compute size configuration options:
+
+- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 3 CUs. A fixed-size compute does not scale to meet workload demand.
```
```suggestion
- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
```

duskpoet commented on content/docs/manage/projects.md:
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
+Neon supports two compute size configuration options:
+
+- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 3 CUs. A fixed-size compute does not scale to meet workload demand.
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 3 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release.
```
I think it's worth to link this blog post for users who are curious how autoscaling works https://neon.tech/blog/postgres-autoscaling

danieltprice replied:
Added!

duskpoet commented:
Please take the screenshot from production, and update 3CUs to 7, that's the current upper limit on prod. Or refrain from pointing it out at all, cause this number will soon change and it may change again in the future

danieltprice commented:
> Please take the screenshot from production, and update 3CUs to 7, that's the current upper limit on prod. Or refrain from pointing it out at all, cause this number will soon change and it may change again in the future

I have replaced the "Pro" screen capture with the Free Tier one, but left information about the compute size options for paid plan users. I also updated content that mentions the max number for CUs.

duskpoet commented on content/docs/manage/projects.md:
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
+Neon supports two compute size configuration options:
+
+- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 3 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release. For more information about N ...
```
```suggestion
- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 7 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release. For more information about Neon's autoscaling feature, see [Scaling serverless Postgres: How we implement autoscaling](https://neon.tech/blog/postgres-autoscaling).
```
